.changeset/seven-rings-yell.md

```diff
@@ -0,0 +1,21 @@
+---
+"thirdweb": minor
+---
+
+Introducing API key restrictions for certain commands. Get your API key from [here](https://thirdweb.com/settings/api-keys).
+
+The following commands will require you to have logged in with your API key:
+
+- `thirdweb deploy`
+- `thirdweb release`
```

Comment:
we shouldn't even mention this, release should tell you already that it's deprecated and to use publish

Reply:
Why do we still have that command if it's deprecated?